teamB/eunsong/app.js

```diff
@@ -0,0 +1,62 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+dotenv.config()
+
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+})
+
+myDataSource.initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  });
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('combined'));
+
+// health check
+app.get("/ping", (req, res) => {
+  res.json({ message : "pong" });
+});
+
+// Create a user
+app.post("/users", async (req, res, next) => {
+  const { name, email, profile_image, password } = req.body
+
+  // console.log(req)
+
+  await myDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      profile_image,
+      password
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [ name, email, profile_image, password ]
+  );
+
+      res.status(201).json({ message : "userCreated" });
```

Comment:
indentation을 잘 맞춰주세요~
```suggestion
  res.status(201).json({ message : "userCreated" });
```